### PR TITLE
salary structure page

### PR DESCRIPTION
**Pull request by Lovepreetcode4each**
1) shifted the salary  structure page from pop to other  page 
2)  completed the edit grade page with new functionality
3) in salary structure show    the  amount upto 2 decimal
4)remove the create button from login 
5) dash board page filled with some static data
6)remove space from text area in salray head

### CODE REVIEW

File: app/Http/Controllers/UsersController.php [hunk 1]
```diff
@@ -69,19 +69,18 @@ public function showListing()
     {
         // Retrieve all users
         $role_id = auth()->user()->role_id;
-       
-        if($role_id == "1"){
+        // admin_access = 1;  
+        if ($role_id == config('app.temp_test_var')) {
```
Comment (kumarisavita) [inline app/Http/Controllers/UsersController.php:73]: name of config variable should represents the value like we are using here for admin_acess so something similar to admin access.

**Review by kumarisavita — CHANGES_REQUESTED** [review]
Please make these small changes and re-submit.

Thanks.

File: app/Http/Controllers/EmployeeController.php [hunk 2]
```diff
@@ -15,117 +15,15 @@ public function index()
     {
         // Get all  employees  came  from success factor  
         //  Join with user detail table 
-        $all_emp = User::with(['post', 'post.get_grade_name'])->where('type_id', '2')->get();
+        $all_emp = User::with('post')->where('type_id', '2')->get()->toarray();
```
[the remark this answers is not in the document]
Comment (Lovepreetcode4each) [reply app/Http/Controllers/EmployeeController.php:18]: done

File: app/Http/Controllers/UsersController.php (hunk 1 above)
Comment (Lovepreetcode4each) [reply app/Http/Controllers/UsersController.php:73]: done

**Review by kumarisavita — APPROVED** [review]

**Pull request merged**